components/script/script_thread.rs

```diff
@@ -1263,10 +1249,10 @@ impl ScriptThread {
             if reports.len() > 0 {
                 path_seg.push_str(", ");
             }
-            path_seg.push_str(current_url);
+            path_seg.push_str(current_url.as_str());
 
             reports.push(Report {
-                path: path![format!("url({})", current_url), "dom-tree"],
+                path: path![format!("url({})", current_url.as_str()), "dom-tree"],
```

Comment:
Oh this is because now `document.url()` return a value while `as_str()` returns a reference, so we have to keep the value on the stack.